disco/extensions/upgrade_simulation/upgrade_simulation.py

```diff
@@ -119,39 +126,44 @@ def run(
+            job_name = self.job.name,
             master_path=self.model.deployment.deployment_file,
             enable_pydss_solve=enable_pydss_solve,
             thermal_config=thermal_config,
             pydss_volt_var_model=pydss_controller_model,
             line_upgrade_options_file=self.get_line_upgrade_options_file(),
             xfmr_upgrade_options_file=self.get_transformer_upgrade_options_file(),
-            thermal_summary_file=self.get_thermal_summary_csv_file(),
+            thermal_summary_file=self.get_thermal_summary_json_file(),
             thermal_upgrades_dss_filepath=self.get_thermal_upgrades_dss_file(),
-            output_csv_line_upgrades_filepath=self.get_line_upgrades_csv_file(),
-            output_csv_xfmr_upgrades_filepath=self.get_transformer_upgrades_csv_file(),
+            output_json_line_upgrades_filepath=self.get_line_upgrades_json_file(),
+            output_json_xfmr_upgrades_filepath=self.get_transformer_upgrades_json_file(),
+            feeder_stats_json_file = self.get_feeder_stats_json_file(),
             verbose=verbose
         )
         determine_voltage_upgrades(
+            job_name = self.job.name,
             master_path=self.model.deployment.deployment_file,
             enable_pydss_solve=enable_pydss_solve,
             pydss_volt_var_model=pydss_controller_model,
             thermal_config=thermal_config,
             voltage_config=voltage_config,
             thermal_upgrades_dss_filepath=self.get_thermal_upgrades_dss_file(),
             voltage_upgrades_dss_filepath=self.get_voltage_upgrades_dss_file(),
-            voltage_summary_file=self.get_voltage_summary_csv_file(),
-            output_csv_voltage_upgrades_filepath = self.get_voltage_upgrades_csv_file(),
+            voltage_summary_file=self.get_voltage_summary_json_file(),
+            output_json_voltage_upgrades_filepath = self.get_voltage_upgrades_json_file(),
+            feeder_stats_json_file = self.get_feeder_stats_json_file(),
             output_folder=self.job_output,
             verbose=verbose
         )
         compute_all_costs(
-            output_csv_xfmr_upgrades_filepath=self.get_transformer_upgrades_csv_file(),
-            output_csv_line_upgrades_filepath=self.get_line_upgrades_csv_file(),
-            output_csv_voltage_upgrades_filepath=self.get_voltage_upgrades_csv_file(),
+            output_json_xfmr_upgrades_filepath=self.get_transformer_upgrades_json_file(),
+            output_json_line_upgrades_filepath=self.get_line_upgrades_json_file(),
+            output_json_voltage_upgrades_filepath=self.get_voltage_upgrades_json_file(),
             cost_database_filepath=cost_database_filepath,
             thermal_cost_output_filepath=self.get_thermal_upgrade_costs_file(),
             voltage_cost_output_filepath=self.get_voltage_upgrade_costs_file(),
             total_cost_output_filepath=self.get_total_upgrade_costs_file()
         )
+    TIMER_STATS.log_stats(clear=True)
```

Comment:
Should this be tabbed to the right once?

Reply:
Sorry, I hadn't pushed my latest commit. Just did